config.md

```diff
@@ -1,13 +1,12 @@
 # <a name="containerConfigurationFile" />Container Configuration file
 
-The container's top-level directory MUST contain a configuration file called `config.json`.
```

Comment:
> this seems like a bizzare flyby

It didn't seem contentious to me, and it's all in the same section of the spec.  But it is an orthogonal change, so I dropped it from this PR with c223790 → 0952dd8 and filed #756 for just the redundant filename removal.